gitlab-data/manage.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+case $@ in
```

Comment:
As you mentioned before, it would be awesome to have tab completions working for this script. (This may just be a future issue)